ee/hogai/assistant.py

```diff
@@ -127,6 +128,7 @@ def __init__(
         new_message: Optional[HumanMessage] = None,
         mode: AssistantMode = AssistantMode.ASSISTANT,
         user: User,
+        session_id: str,
```

Comment:
The Assistant class now runs in different environments, so I'd do like that:
```suggestion
        session_id: str | None = None,
```

I guess the MCP server doesn't really have sessions?